devops/DeployCloudformationS3Bucket/deployStackS3Bucket.py

```diff
@@ -1,32 +1,54 @@
-#Deploys a Stack and S3 Bucket To Cloudformation.
+#Deploys a stack and S3bucket to cloudformation.
+
+#Imports Python libraries
 import boto3
 import re
 import os
 import sys
 import argparse
-
+import random
+import string
+import subprocess
+from datetime import date
 from os.path import dirname
 script_dir = dirname(__file__)
 
-#Creates Cloudformation Client
-cf_client = boto3.client('cloudformation')
+#Creates cloudformation client
+cf_client = boto3.client('cloudformation', region_name='us-west-2')
 
+#Requires user initials input parameter.
 def main(input_initials):
 
-    #Read In Stack Template File
+    #Read in stack template file
     stack_template = _stack_template_file()
 
-    #Parses Input Initials
+    #Parses input initials
     parser = argparse.ArgumentParser(description='Input Initials Required.')
-    parser.add_argument('input_initials', metavar='input_initials', help='Input User Initials For Stack Name.', nargs='?', type=str)
+    parser.add_argument('input_initials', metavar='input_initials', help='Input User Initials To Use In Stack Name.', nargs='?', type=str)
     args = parser.parse_args()
 
-    #Gets Initials
+    #Gets initials
     initials = args.input_initials
     print("\nYour Initials Entered: " + initials + "\n")
```

Comment:
Consider using the `logging` module to create a runtime log instead of printing to the console

Reply:
The print statements are helpful for debugging and running locally, but maybe not best for production. Consider update in the future.